imaging/models.py

```diff
@@ -46,9 +49,11 @@ class TwoPhotonImaging(BaseExperimentalData):
                                  help_text="in nm")
     recording_wavelength = models.FloatField(null=True, blank=True,
                                  help_text="in nm. Can be array for multispectral imaging. TODO: deal with arrays?")
+								 # does django have a way of encoding small arrays in regular tables? I guess not if they are variable size... so it would need to be json
```

Comment:
Yes they can be arbitrary sizes. But you can't have a polymorphic field which is either a Float or an Array. If it's an Array, it's gotta be an Array, which means you have to query it as such; i.e. `SELECT * FROM imaging_twophotonimaging WHERE recording_wavelength[1] = 805;` to look at the first value, or  `SELECT * FROM imaging_twophotonimaging WHERE 805 = ANY(recording_wavelength);` for all values.